### PR TITLE
Update contributing

### PR DESCRIPTION
**Pull request by Oshratn**
Towards #4  CONTIRBUTING.md updated to reflect CNCF template

### CODE REVIEW

**Comment by Oshratn** [comment]
@slashben before approving, please note that I have two open comments in the CONTRIBUTING.md commit that need to be addressed

File: CONTRIBUTING.md
Comment (slashben) [inline CONTRIBUTING.md]: I don't think this is relevant here, it depends on each repo with their own environment setup

File: CONTRIBUTING.md
Comment (slashben) [inline CONTRIBUTING.md]: I am ok with this, we should include these things from the CNCF template (except the one bellow I noted)

**Review by slashben — CHANGES_REQUESTED** [review]

**Comment by Oshratn** [comment]
@slashben completed the requested changes. Ready to merge if you don't see any additional problems.

**Review by slashben — APPROVED** [review]

**Comment by Oshratn** [comment]
@matthyx @rotemamsa I guess I need your approvals too to merge.

**Review by matthyx — APPROVED** [review]

**Review by rotemamsa — APPROVED** [review]

**Pull request merged**
